Userland/Libraries/LibAudio/Loader.cpp

```diff
@@ -80,7 +79,7 @@ LoaderSamples Loader::get_more_samples(size_t samples_to_read_from_input)
 
     if (m_buffer.size() > 0) {
         size_t to_transfer = min(m_buffer.size(), samples_to_read);
-        AK::TypedTransfer<Sample>::move(samples.data(), m_buffer.data(), to_transfer);
+        memmove(samples.data(), m_buffer.data(), to_transfer * sizeof(Sample));
```

Comment:
I think this warrants the introduction of a `move_to` function on Span given the other use cases? That would justify the removal of TypedTransfer from higher levels properly, since they want/need a move primitive in some cases.

Reply:
"move_to" would imply that the span is "moved from" which to me means that the input span is zeroed. Is that the intent of this line of code?

Reply:
Hm. For this code, it doesn't matter. I guess it can be changed to a copy, but we might still need a move function in the future (not in this PR).